pytests/tests/test_downgrade.py

```diff
@@ -22,15 +22,8 @@ def teardown_test(utils):
 
 
 def test_downgrade_no_arg(utils):
-    ret = utils.run(['tdnf', '--config', '/etc/tdnf/tdnf.conf', '--enablerepo=photon-debuginfo', '--assumeno', 'downgrade'])
-    try:
-        if os.environ['DIST'] != 'photon':
-            assert(ret['retval'] == 1602)
-        else:
-            assert(ret['retval'] == 8)
-    except Exception as e:
-        # Default to Photon
-        assert(ret['retval'] == 8)
+    ret = utils.run(['tdnf', 'downgrade'])
```

Comment:
The reason I used different config for Photon was, we install glibc-debuginfo in test environment and with that 'tdnf downgrade' should try to downgrade everything and with assume assumeno we should get 1602 as retval.
TBH, I didn't knew what 'tdnf downgrade' without args will do and @keerthanakalyan gave me insight on how it is supposed to work so I tried to achieve it.

Reply:
Ohh, I see. Then we must probably call it `test_downgrade_everything` and create some new dummy spec files that simulate this situation. For now, let the `test_downgrade_no_args` be consistent with it's naming (with other such tests everywhere). We'll put in a TODO to add the other test.